code/modules/projectiles/guns/mounted.dm

```diff
@@ -457,6 +457,13 @@
 	. = ..()
 	sponson = new sponson(src)
 
+/obj/machinery/deployable/mounted/moveable/atgun/attackby(obj/item/I, mob/user, params)
+	if(user.interactee == src)
+		balloon_alert(user, "Cannot reload while you're manning it")
```

Comment:
Oh this is also not checking for what I is. So the balloon alert doesn't really make sense if you're smacking it with some other item.